Python 3.14.0 (tags/v3.14.0:ebf955d, Oct  7 2025, 10:15:03) [MSC v.1944 64 bit (AMD64)] on win32
Enter "help" below or click "Help" above for more information.

= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
Traceback (most recent call last):
  File "C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py", line 1, in <module>
    import telebot
ModuleNotFoundError: No module named 'telebot'
>>> 
= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
Traceback (most recent call last):
  File "C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py", line 2, in <module>
    import httpx
ModuleNotFoundError: No module named 'httpx'
>>> 
= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
Traceback (most recent call last):
  File "C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py", line 9, in <module>
    bot = telebot.TeleBot(TELEGRAM_TOKEN)
  File "C:\Users\SHAHZOD\AppData\Local\Programs\Python\Python314\Lib\site-packages\telebot\__init__.py", line 196, in __init__
    util.validate_token(self.token)
  File "C:\Users\SHAHZOD\AppData\Local\Programs\Python\Python314\Lib\site-packages\telebot\util.py", line 698, in validate_token
    raise ValueError('Token must contain a colon')
ValueError: Token must contain a colon
>>> 
= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
ðŸ¤– Bot ishga tushdi...

= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
ðŸ¤– Bot ishga tushdi...

= RESTART: C:/Users/SHAHZOD/AppData/Local/Programs/Python/Python314/groq_telegram_bot.py
ðŸ¤– Bot ishga tushdi...
